src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/postsubmit/RequestFeeRemissionConfirmationTest.java

```diff
@@ -0,0 +1,94 @@
+package uk.gov.hmcts.reform.iacaseapi.domain.handlers.postsubmit;
+
+import static junit.framework.TestCase.assertTrue;
+import static org.assertj.core.api.Assertions.*;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.Assert.assertFalse;
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.reset;
+import static org.mockito.Mockito.when;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.AsylumCase;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.ccd.Event;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.ccd.callback.Callback;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.ccd.callback.PostSubmitCallbackResponse;
+
+@ExtendWith(MockitoExtension.class)
+@SuppressWarnings("unchecked")
+class RequestFeeRemissionConfirmationTest {
+
+    @Mock private Callback<AsylumCase> callback;
+
+    private RequestFeeRemissionConfirmation requestFeeRemissionConfirmation;
+
+    @BeforeEach
+    void setUp() {
+
+        requestFeeRemissionConfirmation = new RequestFeeRemissionConfirmation();
+    }
+
+    @Test
+    void should_return_the_confirmation() {
+
+        when(callback.getEvent()).thenReturn(Event.REQUEST_FEE_REMISSION);
+
+        PostSubmitCallbackResponse callbackResponse =
+            requestFeeRemissionConfirmation.handle(callback);
+
+        assertNotNull(callbackResponse);
+        assertTrue(callbackResponse.getConfirmationHeader().isPresent());
+        assertTrue(callbackResponse.getConfirmationBody().isPresent());
+
+        assertThat(callbackResponse.getConfirmationHeader().get())
+            .contains("# Your remission request has been submitted");
+        assertThat(callbackResponse.getConfirmationBody().get())
+            .contains("#### What happens next\n\n"
+                      + "The Tribunal will review the remission request. The decision details will be available to view in the appeal tab "
+                      + "when the decision has bee made.<br>");
```

Comment:
Typo correction above should reflect here

Reply:
removed